x/gov/queryable.go

```diff
@@ -171,7 +171,7 @@ type QueryProposalsParams struct {
 	Voter              sdk.AccAddress
 	Depositer          sdk.AccAddress
 	ProposalStatus     ProposalStatus
-	NumLatestProposals int64
+	NumLatestProposals uint64
```

Comment:
uint64 is fine but I would just call this "Limit" or "Count", and assume that it will return the latest X.